### ISSUE TITLE
artwork

### ISSUE THREAD

**Issue by udeved** [0ADMods/HanChina#3]
Artwork tasks (whoever take it on)

* government center should be rescaled to use less space, a wee bit smaller foundation
* two variants of artillery towers
* a siege ram
* heads, helmets, weapons, shield and hero rework, eg the crown is too small for the horse hero

**Comment by udeved** [comment]
A dao sword model would be very much appreciated.

https://static.turbosquid.com/Preview/000330/900/OC/dao-sword-3d-3ds_DHQ.jpg

**Comment by StanleySweet** [comment]
The sword comes from a later period I believe. All the Han Dynasty swords I've seen so far are straight.
> government center should be rescaled to use less space, a wee bit smaller foundation
Why ?
> heads, helmets, weapons, shield and hero rework, eg the crown is too small for the horse hero
What about the rest?

I believe it's better to create individual tasks instead of creating only one issue. Like it's currently done in terra_magna.

> check animations of cavalry bow and rcrossbow
> animate the liannu crossbow?
@AlexanderVzla  is on that

**Comment by udeved** [comment]
@StanleySweet 

The gov center is huge on smaller maps with less space. I find it simply too big foundation, it eats too much space. Perhaps little smaller than the fortress?

Agreed on specific issues.

I tried to correct the cav bow animation. What are the load & event values for? 

Afaik, the Dao is a cavalry broadsword, saber like, that came in use during Han dynasty.

**Comment by StanleySweet** [comment]
The problem is I already reduced it I'm not sure how much I can still squeeze it...
Before anything else I'd like for this mod to be a dep of terra magna. I'd rather not have to work on too fronts :)

They are for sound I believe.
What you are looking for should be in the template.


Well there are a few swords labelled daos in the mod and they were depicted as long straight swords. Han dynasty dao on google image also reveals the same

**Comment by udeved** [comment]
@StanleySweet 

Let's make this mod the default working mod, where people commit Han stuff.

I created a Xiongnu repo, I'll fill it with xiongnu in the coming weeks if time allows, should also become default repo. I won't have any time for meso american factions, but I would like to stick with anything East Asia only from TM.

A question.

I made basically clothes variants for the Han. I found these huge art folders with files named {civ}_* quite uhm, ugly, Is there a gmae engine restriction(hardcoded) paths in art?
What if sub folders were used instead of the {civ}_* prefix? Would contribute to readability and encapsulate civ specific things. Shared could go in generic folder similar as already partially introduced.

Regarding the Dao, if you read a bit on the Dao, they write the curved Dao developed during Han and became default.

**Comment by StanleySweet** [comment]
> Default working mod.
Sure that's the goal. Thanks for your time and energy.
> TM = Asian Civ Only
I disagree but that's not important right now. The important part is reducing the coupling between civs by creating folders for them. They can then all be added as dependencies. That's a feature tha'ts currently underused.

Maybe a rise of the east mod can be recreated at some point.
> Naming conventions.
Subfolders can be used. You can push it to the limits like I did for the stella_artis mod so it's a good idea to do it. Once again wasn't done because like of time.

The only important thing is that any renaming matches the naming conventions as defined in the wiki 
https://trac.wildfiregames.com/wiki/ArtFileNamingConventions
Example part should not be followed.

**Comment by udeved** [comment]
One more question @StanleySweet 

What do we do with maps, map props etc? I think these should go in whatever meta mod will be used. Currently, two tarim maps are in the Han repo... and xiongnu have to be decoupled from Han props, meshes etc.

**Comment by StanleySweet** [comment]
Maps belong to terra magna if they have multiple civ in them. Vegetation should be integrated in the main game.
The rest should go in their respective mods.

We need to make a list of objects that are dependencies for xiongnu ( we can make han_china a dependency for the time being and create a task to decouple the civs.

**Comment by udeved** [comment]
@StanleySweet 

I have started copying xiongnu in the new repo.

There is already one, the zebu needs be likely in the main public mod.

* zebu ( shared by Han and Xiongnu)
* bamboo
* Han bridge & gate

I am not sure about the tons of various chinese fields in TM, but I would find a rice variant foield attractive on cerrtain biome.

**Comment by StanleySweet** [comment]
Zebu is in the public mod. 

The rice fields were for han_china. There is currently way no way to change the field depending on the biome.

As for Dadao (the variant of the dao you are talking about I couldn't find a strong reference as it being used before the XXth century.

**Comment by udeved** [comment]
I suggest we make a rise of the east repo that contains maps, props, objects for xiongnu and Han? We can't use the TM ones, the maps in Han are modified to work.

**Comment by StanleySweet** [comment]
> Creating a "rise_of_the_east_mod"
Sure go ahead. It should have "han_china" and "xiongnu" as dependencies. Still not sure what to do with the GUI assets. I believe current maps do not depend on XIONGNU at all and therefore should go in the "han_china" mod.

**Comment by udeved** [comment]
Yep, maps have a Han depend only, but tarim maps make most sense with Han vs Xiongnu or Seleucids eventually. There is also a scenario map iirc that uses the Han bridge and gate vs another chinese player.

**Comment by StanleySweet** [comment]
Well the goal is to limit coupling. So people can play "rise_of_the_east" with maps from "han_china"

**Comment by udeved** [comment]
Pretty much my idea too. :+1:

**Comment by StanleySweet** [comment]
Good!

**Comment by udeved** [comment]
You might find this interesting...

https://www.youtube.com/watch?v=Hb4vEMGWccc
